### PR TITLE
APP-2782 Fixed an issue where resizeObserver was non existent on Safari

### PR DESCRIPTION
**Pull request by joanofdart**
Related to #2782 
* ResizeObserver issue.

### CODE REVIEW

File: libs/ui/src/lib/layout/layout.module.ts [hunk 1]
```diff
@@ -35,22 +35,19 @@ export class Layout implements OnInit, OnDestroy {
...
-  ngOnInit() {
+  async ngOnInit() {
     const el = this.el.nativeElement;
-    if (ResizeObserver) {
-      this.observer = new ResizeObserver(([entry]) => this.width.next(entry.contentRect.width));
-      this.observer.observe(el);
-    } else {
-      // do something with window:resize
-    }
+    const resizeObs = (ResizeObserver in window) ? (await import('@juggle/resize-observer')).ResizeObserver : ResizeObserver;
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/layout/layout.module.ts:40]:
```typescript
if (!window) {
  return;
}
const resizeObs = ResizeObserver in window) ? ...
```

File: libs/utils/src/lib/media/media.firestore.ts [hunk 2]
```diff
@@ -37,7 +37,6 @@ export function getImgSize(url: string) {
   } else if (url.includes('still')) {
     return { original: 0, xs: 50, md: 100, lg: 200 };
   } else {
-    console.warn('No bucket directory, exiting function');
-    return false;
+    throw new Error('No bucket directory, exiting function');
```
Comment (phpgeekfr) [inline libs/utils/src/lib/media/media.firestore.ts:40]: This message does not seems very relevant

File: libs/ui/src/lib/layout/layout.module.ts [hunk 3]
```diff
@@ -35,22 +35,19 @@ export class Layout implements OnInit, OnDestroy {
...
+    this.observer = new resizeObs(([entry]) => this.width.next(entry.contentRect.width));
+    this.observer.observe(el);
```
Comment (phpgeekfr) [inline libs/ui/src/lib/layout/layout.module.ts:42]: LGTM but wait for @GrandSchtroumpf point of view

File: libs/utils/src/lib/media/media.firestore.ts (hunk 2 above)
Comment (joanofdart) [reply libs/utils/src/lib/media/media.firestore.ts:40]: Any idea on what message should be there instead? :)

File: libs/ui/src/lib/layout/layout.module.ts (hunk 3 above)
Comment (joanofdart) [reply libs/ui/src/lib/layout/layout.module.ts:42]: LGTM? ahh looks good to me?

File: libs/utils/src/lib/media/media.firestore.ts (hunk 2 above)
Comment (joanofdart) [reply libs/utils/src/lib/media/media.firestore.ts:40]: @phpgeekfr

**Pull request merged**